tests/opentelemetry/test_span.py

```diff
@@ -95,6 +95,66 @@ def test_otel_span_kind(oteltracer):
         pass
 
 
+@pytest.mark.snapshot(wait_for_num_traces=1)
+@pytest.mark.parametrize("span_kind", [OtelSpanKind.CLIENT, OtelSpanKind.SERVER])
```

Comment:
Thanks for the fix! 

Can you add a fix release note as well. 

Here's the docs on how to generate and write a release note: https://github.com/DataDog/dd-trace-py/blob/main/docs/releasenotes.rst#generating-a-release-note.